### PR TITLE
Copy edits of full state, estimator, and toffoli simulator articles

### PR DESCRIPTION
**Pull request by Bradben**
* Copy edit for Azure standards
* Some dev edits for SEO, readability, and consistency 
* Updates to reflect Python, Cmd line and Jupyter support.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/machines/full-state-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/3bc1d7b89e795667dd80ad23a3e92ca9743d55a9/articles/user-guide/machines/full-state-simulator.md) | 84 | 59 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca382e30-b673-4c12-872d-982278d0bf7a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/resources-estimator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/3bc1d7b89e795667dd80ad23a3e92ca9743d55a9/articles/user-guide/machines/resources-estimator.md) | 87 | 55 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01c3e891-0d93-4ee0-8740-1a6c17bad324_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/toffoli-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/3bc1d7b89e795667dd80ad23a3e92ca9743d55a9/articles/user-guide/machines/toffoli-simulator.md) | 81 | 49 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22e8c716-239a-450e-b92a-6627fc2a154a_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/machines/full-state-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1613cc6a7a5f439608d31447f5f3bda39921a60/articles/user-guide/machines/full-state-simulator.md) | 84 | 59 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4fedeaa-396c-4ee1-81a2-d13d9ab3cb82_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/resources-estimator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1613cc6a7a5f439608d31447f5f3bda39921a60/articles/user-guide/machines/resources-estimator.md) | 87 | 55 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d535c10f-0d38-4a13-9222-48b14e8076f2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/toffoli-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1613cc6a7a5f439608d31447f5f3bda39921a60/articles/user-guide/machines/toffoli-simulator.md) | 81 | 49 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/819064ba-c86d-458b-868b-e77e67136318_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/user-guide/machines/full-state-simulator.md [hunk 1]
```diff
@@ -28,19 +28,37 @@ of the quantum operation you want to execute along with the rest of the paramete
...
+```
+
+### Invoking the simulator from Juptyer Notebooks
+
+Use the IQ# magic command [%simulate](xref:microsoft.quantum.iqsharp.magic-ref.simulate) to run the Q# operation.
+
+```dotnetcli
+%simulate myOperation
+```
+## Seeding the simulator
```
[the remark this answers is not in the document]
Comment (rmshaffer) [reply articles/user-guide/machines/full-state-simulator.md:59]: Currently it's not possible. Feel free to file an issue in the IQ# repo if that's something that would be desirable!

**Comment by Bradben** [comment]
> @Bradben These are great changes, thanks again. Just a bunch of smaller things, but I did notice one overarching thing (inherited from the old versions): the capitalization of "Full State Simulator", "Trace Simulator", etc. is inconsistent across (and within) the pages. I'm not sure which direction it needs to go, but the inconsistency really stood out

Thanks - I made them consistently lower case across the doc set.

File: articles/user-guide/machines/full-state-simulator.md [hunk 2]
```diff
@@ -28,19 +28,37 @@ of the quantum operation you want to execute along with the rest of the paramete
...
+```python
+qubit_result = myOperation.simulate()
+```
+
+### Invoking the simulator from the command line
 
-The `QuantumSimulator` class implements <xref:System.IDisposable>, thus the `Dispose` method
-should be called once the instance of the simulator is not used anymore. The best way 
-to do this is to wrap the simulator within a `using` statement, as in the example above.
+When running a Q# program from the command line, the Full State Simulator is the default target machine. Optionally, you can use the *--simulator* (or *-s* shortcut) parameter to specify the desired target machine. Both of the following commands run a program using the Full State Simulator.
```
[the remark this answers is not in the document]
Comment (Bradben) [reply articles/user-guide/machines/full-state-simulator.md:43]: Actually, as command line parameters, they are supposed to be bold :-)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/machines/full-state-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/07961e35e0741cf8bd324e59c4f34f5891decf03/articles/user-guide/machines/full-state-simulator.md) | 85 | 54 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f2795c04-513e-49e5-a9a6-2e06a7193175) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/resources-estimator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/07961e35e0741cf8bd324e59c4f34f5891decf03/articles/user-guide/machines/resources-estimator.md) | 83 | 52 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/81bc3cc1-7e39-40b9-8612-980ba416fd84) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/toffoli-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/07961e35e0741cf8bd324e59c4f34f5891decf03/articles/user-guide/machines/toffoli-simulator.md) | 81 | 46 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f4282a30-c940-445d-affe-a0e2244e4f0c) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by Bradben** [comment]
@gillenhaalb - Thanks, should be good now.

File: articles/user-guide/machines/full-state-simulator.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gillenhaalb) [reply articles/user-guide/machines/full-state-simulator.md]: Ohhh my mistake, thanks for the heads up!

**Comment by gillenhaalb** [comment]
@Bradben The changes are great, but I spotted something I missed on the first go. On the resource estimator and Toffoli pages where the implementations (cmd line, Python, C#, Jupyter) are briefly presented, the language is accidentally `dotnetcli` for the Jupyter code block. Since Jupyter isn't directly supported, I figure just removing the `dotnetcli` will do it.

I can't add it as a suggestion (GitHub interprets the resulting plain triple tick incorrectly) but the lines are 83 and 54 for the estimator and Toffoli pages, respectively. 

Also there's an accidental extra line in the command line code blocks on the same pages (lines 76, 47 resp.)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/machines/full-state-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e8542e1c15db32130385e6579c7ff6ec8b0eef2e/articles/user-guide/machines/full-state-simulator.md) | 85 | 54 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/211af931-9a17-405c-8b2f-f7197a34c28f) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/resources-estimator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e8542e1c15db32130385e6579c7ff6ec8b0eef2e/articles/user-guide/machines/resources-estimator.md) | 83 | 52 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e7eabb2-c3bf-4223-b54a-21a757c828e4) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/machines/toffoli-simulator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e8542e1c15db32130385e6579c7ff6ec8b0eef2e/articles/user-guide/machines/toffoli-simulator.md) | 81 | 46 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/635caebb-fc96-4d5c-82ab-0242a41c60b1) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by Bradben** [comment]
@gillenhaalb - thanks, should be good now.

**Pull request merged**